feedback/README.md

```diff
@@ -0,0 +1,6 @@
+This folder contains the Google Apps Script that feeds feedback responses
+through a Google Form and stores them in its linked spreadsheet, along with the
+entered client information at the time of the response. It is managed locally
```

Comment:
OK I'll change that part (or you can just do it, since I won't be available till tonight).

As for the line breaks, I did that to make it easier to read without word wrap, and Markdown ignores single line breaks, right? You can change it if you hate it though.

Reply:
Heh, forgot markdown ignores single line breaks. No worries, was mostly just curious about the choice.